scio-jdbc/src/it/scala/com/spotify/scio/jdbc/PopulateTestData.scala

```diff
@@ -0,0 +1,41 @@
+package com.spotify.scio.jdbc
+
+import com.spotify.scio.jdbc.sharded.JdbcUtils
+
+import java.sql.DriverManager
```

Comment:
You don't use this import here I think

Reply:
Thanks. Looks our scalafix is not running in the `IntegrationTest` config since this has not been spotted by our CI.